src/client/src/ui/spotTile/model/spotTileModel.js

```diff
@@ -99,6 +106,13 @@ export default class SpotTileModel extends ModelBase {
     this._regionManagerHelper.popout(`${this.currencyPair.symbol} Spot`, 332, 155);
   }
 
+  @observeEvent('openChartIQ')
+  _onOpenChartIQ(){
```

Comment:
can we rename all usages of 'chart iq' with the actually functionality? Like `displayCurrencyChart`?